### DISCUSSION TITLE
[v4] CLI equivalent to `tailwindcss init`

### DISCUSSION THREAD

**Discussion by petecooper** [tailwindlabs/tailwindcss#15252] [Help] [answered]
First time caller, I'm going in on the v4 beta for a baptism of fire. I'm using the standalone CLI. No Node (as yet).

The docs for v3 state that `tailwindcss init` is used to initialise a new instance. That command doesn't work with v4.0.0 beta 4:

```
$ tailwindcss init
≈ tailwindcss v4.0.0-beta.4

Invalid command: init

Usage:
  tailwindcss [options]

Options:
  -i, --input ··········· Input file
  -o, --output ·········· Output file
  -w, --watch ··········· Watch for changes and rebuild as needed
  -m, --minify ·········· Optimize and minify the output
      --optimize ········ Optimize the output without minifying
      --cwd ············· The current working directory [default: `.`]
  -h, --help ············ Display usage information
```

The v4 beta docs (https://tailwindcss.com/docs/v4-beta) allude to it being possible:

>You can also download [standalone builds](https://github.com/tailwindlabs/tailwindcss/releases/tag/v4.0.0-beta.1) of the new CLI tool from GitHub for projects that don’t otherwise depend on the Node.js ecosystem.

Is there a workaround for this, please? Or do I need to go down the Node route for now?

Thank you.

**Comment by adamwathan** [comment, marked as answer]
Hey! With v4 there's nothing really to init because there's all configuration is optional. Just create a CSS with `@import "tailwindcss";` in there somewhere, build that file with the CLI tool and you're good to go.

**Reply by petecooper** [reply]
Perfect. Thanks @adamwathan - you're doing fine work here, please know it is greatly appreciated.
